### ISSUE TITLE
Moderate content: Administrator

### ISSUE THREAD

**Issue by Avina20** [gcivil-nyu-org/wed-fall24-team4#2]
An admin should be allowed to moderate all user-generated content, including comments, status updates about stations, and accessibility features. He should be able to flag, hide, or delete the inappropriate or misleading content, respond to user comments or reports, both publicly and privately and block or restrict users who repeatedly violate platform policies.
